src/main/java/rife/bld/dependencies/ArtifactRetriever.java

```diff
@@ -189,7 +189,13 @@ public boolean transferIntoDirectory(RepositoryArtifact artifact, File directory
     }
 
     private static URLConnection openUrlConnection(RepositoryArtifact artifact) throws IOException {
-        var connection = new URL(artifact.location()).openConnection();
+        String location;
```

Comment:
This is probably not what you wanted to commit with the rest

Reply:
Yeah, thanks. Fixed now.